tests/admin/test_controller.py

```diff
@@ -1442,6 +1445,10 @@ def test_libraries_post_edit(self):
                 Configuration.ENABLED_FACETS_KEY_PREFIX + FacetConstants.ORDER_FACET_GROUP_NAME,
                 library).value)
 
+        # A library-wide setting that was not updated has been left alone.
```

Comment:
I would clarify this comment since it only applies to images. Other setting types will be removed if they're not included in the edit request.